tools/ingest-converter/src/test/resources/org/logstash/ingest/logstashDotsInConvertField.conf

```diff
@@ -5,3 +5,8 @@ filter {
       }
    }
 }
+output {
+   elasticsearch {
+      hosts => "localhost"
```

Comment:
@original-brownbear yep, it is, but I wanted to be explicit and add this so it can be used as a template for users. So if someone has ES on a different host, they can easily change the value with this.